tools/cli/templates/projects/vite/README.md

```diff
@@ -0,0 +1,31 @@
+# The default coherent-guic-cli Vite template
+
+This is the default template for [GameUIcomponents](https://coherentlabs.github.io/GameUIComponents/en/) using [Vite](https://vitejs.dev/) creates using [coherent-guic-cli](https://www.npmjs.com/package/coherent-guic-cli).
+If you need to change Vite's configuration - create a vite.config.js in the root and setup whatever options you need to change. Check the full configuration guide [here](https://vitejs.dev/config/).
+
+## How to Run
+
+Run
+```
+coherent-guic-cli create vite <directory>
+```
+
+After the project has been created and its dependencies installed, run:
+```
+cd <directory>
+```
+
+and use one of the following commands:
+
+- will start a development server with HMR and page reload (no need for polyfills)

```

Comment:
I assume that if I am reading this README, the template is already created. Maybe we should remove it and only have how to run the created project, not how to create it?

Reply:
yes, but I was thinking if someone's looking at the template's readme in Github like this one - https://github.com/kometolabs/vite-tailwind-nojs-starter